### PR TITLE
CI and Sandbox: Temporarily remove use of `uv`

### PR DESCRIPTION
**Pull request by amotl**
`uv` provided some hiccups in dependency resolutions, so we are trying without, in order to check if this improves the situation, or if it's unrelated.

- GH-32
- GH-35

Most prominently, the update to a higher version of pandas still does not succeed after many cycles to make it so.

- GH-27

### CODE REVIEW

**Comment by charliermarsh** [comment]
So sorry to bother but for my own prioritization, are you able to share the resolution that didn't succeed? Is it:

```
cratedb-toolkit[datasets]==0.0.29
ipyleaflet<0.20
kaleido<0.3
llvmlite>=0.40
pandas<2.1
pandas<2.3
plotly<5.25
pycaret>=3,<3.4
pydantic<3
refinitiv-data<1.7
sqlalchemy<2.1
sqlalchemy-cratedb==0.40.1
```

**Comment by amotl** [comment]
Dear Charlie,

thanks a stack for your reply, and sorry for missing it.

> for my own prioritization

TLDR; Any issues on our ends have been resolved well. @zanieb educated us about https://github.com/astral-sh/uv/issues/8157, but I am no longer sure this was even the problem here. Thank you both for your time and diligence on supporting us and tracking this back.

With kind regards,
Andreas.


### Details

The issue has been resolved and we swapped `uv` back in per GH-39. The problem originated because of the constraint limit `pandas<2.1`, which, when resolved per GH-27, did no longer cause any issues, afair.

> able to share the resolution that didn't succeed?

Because you are listing `pandas<2.1`, I guess your enumeration is on the right track. Please note, however:

- You are listing pandas twice, as you also included `pandas<2.3`.
- Please also note we added and changed constraints about llvmlite, pycaret, and kaleido while trying to resolve woes on CI, see GH-29, GH-32, GH-33, GH-35.

### Conclusion

Wrapping up, I think the noise was more about our own sillyness than about a serious issue we observed with uv. If you think our constellation of dependencies can support you to work on https://github.com/astral-sh/uv/issues/8157, please let us know any time. We can easily travel back in time of the repository history to get you a dependency dump using `pipdeptree`, or any other incantation of `uv` with relevant debug options you may advise.
